### DISCUSSION TITLE
Prepare for x86_64-v3 repos

### DISCUSSION THREAD

**Discussion by wszqkzqk** [msys2/MINGW-packages#16820] [Ideas]
[Arch Linux developers discussed to start x86_64-v3 port](https://lists.archlinux.org/archives/list/arch-dev-public@lists.archlinux.org/thread/5QEXAPTOI5KGHHCWXGFS4AT2IMZQ7IJE/), openSUSE has [launched this](https://www.phoronix.com/news/openSUSE-TW-x86-64-v3-HWCAPS) and Fedora also has planed to [add x86-64 v2/v3/v4 feature levels](https://www.phoronix.com/news/Fedora-39-RPM-4.19). Is it necessary for MSYS2 to add x86_64-v3 feature level for performance?

**Comment by MehdiChinoune** [comment]
I proposed that before, they had a discussion about it in a previous meeting.
Most of maintainers have no interest on the idea.

see: https://pads.c3w.at/code/#/2/code/edit/oXdtR5THJy75DZIllZBAnEHw/ -> 2021-11-07
```
MinGW-x86_64-v3?
- minimum CPU level in our mingw repos should cover what supported Windows version covers; once we drop support of some Windows version, we can bump CPU level accordingly
- doesn't seem like worth the effort to maintain separate package repo
- we can recommend the user to maintain an unofficial repo with just the packages that are really faster with v3 and we can link to it on msys2.org for speed enthusiasts :)
```

They maintain two separate repos for expired/outdated/non-used 32bit arch, but couldn't maintain a repo for  ~70% (at least) of users!
